Library/Homebrew/formula_installer.rb

```diff
@@ -337,6 +337,10 @@ def build_bottle_postinstall
 
   sig { void }
   def install
+    # TODO: Remove when bottles are built with RPATH relocation enabled
+    # https://github.com/Homebrew/brew/issues/11329
+    ENV["HOMEBREW_RELOCATE_RPATHS"] = "1"
```

Comment:
Should this be done in `pour` perhaps instead (ideally as close as possible to the relevant call site)? Also, need to make sure it gets unset after it's no longer needed. `with_env` may be able to help with that.

Reply:
Good suggestion. Done.